lib/utapi/utilities.js

```diff
@@ -103,9 +102,10 @@ export function listMetrics(metricType) {
         .option('-p, --port <port>', 'Port of the server')
         .option('--ssl', 'Enable ssl')
         .option('-v, --verbose')
+        .option('-r, --recent', 'List metrics of the latest 15 minute interval')
```

Comment:
this is the last full 15 minute interval?  not from 2:15 to current if it is 2:20 for instance?  please clarify